ast/shared/src/main/scala/jawn/ast/JValue.scala

```diff
@@ -81,6 +82,7 @@ sealed abstract class JValue {
 
 object JValue {
   implicit val facade: Facade[JValue] = JawnFacade
+  implicit private[jawn] lazy val classTag: ClassTag[JValue] = ClassTag(classOf[JValue])
```

Comment:
note that `lazy` adds some additional synchronization overhead compared to a vanilla `val`, which is why we have unsafe lazies in Scala 3.

Reply:
I suppose it could be rewritten as a null init and probably squeeze out a few more femtoseconds.

Reply:
Re: `val` vs `lazy val`, I didn't confirm with a benchmark, but my speculation was that `val` would regress the load time for users who just see `JValue` as type signature but do not hit the UnwrapArray parser.
https://github.com/typelevel/jawn/pull/547#discussion_r1233129202

Reply:
Right, sure :) basically we are trading off a one-time load penalty for a penalty on every access.